bins/more/sim.fzb

```diff
@@ -158,109 +157,33 @@
-        <instance modelIndex="38" moduleIdRef="4001_4_x_2_input_NOR_gate_multipart">
+        <!--
+        <instance moduleIdRef="9f2b90fcf9c65c9bc09d570e3442069e" modelIndex="28" path="solenoid.fzp"><views><iconView layer="icon"><geometry z="-1" x="-1" y="-1"/></iconView></views></instance><instance moduleIdRef="1099ABDE24_ST_relay" modelIndex="29" path="basic-relay.fzp"><views><iconView layer="icon"><geometry z="-1" x="-1" y="-1"/></iconView></views></instance><instance moduleIdRef="71d4db74a3a0ec43d9303ddffee7361b" modelIndex="29" path="DIP_Relay_D31A.fzp"><views><iconView layer="icon"><geometry z="-1" x="-1" y="-1"/></iconView></views></instance>-->
+        <instance modelIndex="3" moduleIdRef="__spacer__" path="ICs">
             <views>
                 <iconView layer="icon">
                     <geometry x="-1" y="-1" z="-1"/>
                 </iconView>
             </views>
         </instance>
-        -->
+        <!-- 
+        <instance modelIndex="33" moduleIdRef="a59353bd0db0225dc22770e292c622f3" path="lm358.fzp"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance><instance modelIndex="34" moduleIdRef="1001AAAB555Timer" path="555timer.fzp"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance><instance modelIndex="35" moduleIdRef="1001AAACOptocoupler" path="optocoupler.fzp"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance><instance modelIndex="36" moduleIdRef="a4f38a10-dd92-11dd-a3e2-001f5b3a17a4" path="h-bridge.fzp"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance><instance modelIndex="36" moduleIdRef="1a7828b6466fac9fd042221e931124c4" path="74HC595.fzp"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance><instance modelIndex="36" moduleIdRef="HEF4094" path="HEF4094.fzp"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance><instance moduleIdRef="RevEd_FRM050"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance><instance moduleIdRef="RevEd_ULN2003A"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance><instance modelIndex="38" moduleIdRef="CrystalModuleID" path="crystal.fzp"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance><instance modelIndex="38" moduleIdRef="37asdf3c843a2127a0e499f8a0b3ef6aCrystalModuleID" path="resonator-3pin.fzp"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance><instance modelIndex="38" moduleIdRef="4001_4_x_2_input_NOR_gate_single"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance><instance modelIndex="38" moduleIdRef="4001_4_x_2_input_NOR_gate_multipart"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance>-->
```

Comment:
Are the line breaks intentionally removed? Since the section is commented out, maybe it can entirely be removed?

Reply:
My tool for auto-indent removes the line breaks for the commented lines. These lines are commented as these parts do not have SPICE fields currently, but they could be added more or less easily. It is a kind of TODO list for me, but we can remove them if you prefer it.

Reply:
Ah, great. I am working with someone to add more SPICE fields right now. The it would make sense to start with those parts, right?

Reply:
Yes. Stating with these ones makes sense. However, there are a few that will not be useful until we extend the simulator to work with transitory analysis, such as the 555, the HEF4094 or the 74HC595.  Maybe they could be a second step. 

I have a functional example of an NAND gate, but I would like to add some LEDs at the input and output to show the signals. But my model does not handle well the LEDs (the NAND gate cannot drive the LEDs). Maybe I could send it to you and the person can use it as a starting point.